static/lang/re-en.json

```
@@ -1193,252 +1193,252 @@
-                "success": "[[/r 1d4[persistent,mental]]]"
+                "criticalSuccess": "@Damage[(2*1d4)[persistent,mental]]",
+                "success": "@Damage[1d4[persistent,mental]]"
             },
             "Mental1d6": {
-                "criticalSuccess": "[[/r (2*1d6)[persistent,mental]]]",
-                "success": "[[/r 1d6[persistent,mental]]]"
+                "criticalSuccess": "@Damage[(2*1d6)[persistent,mental]]",
+                "success": "@Damage[1d6[persistent,mental]]"
             },
             "Mental1d8": {
-                "criticalSuccess": "[[/r (2*1d8)[persistent,mental]]]",
-                "success": "[[/r 1d8[persistent,mental]]]"
+                "criticalSuccess": "@Damage[(2*1d8)[persistent,mental]]",
+                "success": "@Damage[1d8[persistent,mental]]"
             },
             "Mental2d6": {
-                "criticalSuccess": "[[/r (2*2d6)[persistent,mental]]]",
-                "success": "[[/r 2d6[persistent,mental]]]"
+                "criticalSuccess": "@Damage[(2*2d6)[persistent,mental]]",
+                "success": "@Damage[2d6[persistent,mental]]"
             },
             "Mental3d6": {
-                "criticalSuccess": "[[/r (2*3d6)[persistent,mental]]]",
-                "success": "[[/r 3d6[persistent,mental]]]"
+                "criticalSuccess": "@Damage[(2*3d6)[persistent,mental]]",
+                "success": "@Damage[3d6[persistent,mental]]"
             },
             "Negative1d10": {
-                "criticalSuccess": "[[/r (2*1d10)[persistent,void]]]",
-                "success": "[[/r 1d10[persistent,void]]]"
+                "criticalSuccess": "@Damage[(2*1d10)[persistent,void]]",
+                "success": "@Damage[1d10[persistent,void]]"
             },
             "Negative1d6": {
-                "criticalSuccess": "[[/r (2*1d6)[persistent,void]]]",
-                "success": "[[/r 1d6[persistent,void]]]"
+                "criticalSuccess": "@Damage[(2*1d6)[persistent,void]]",
+                "success": "@Damage[1d6[persistent,void]]"
             },
             "Negative1d8": {
-                "criticalSuccess": "[[/r (2*1d8)[persistent,void]]]",
-                "success": "[[/r 1d8[persistent,void]]]"
+                "criticalSuccess": "@Damage[(2*1d8)[persistent,void]]",
+                "success": "@Damage[1d8[persistent,void]]"
             },
             "Negative2d10": {
-                "criticalSuccess": "[[/r (2*2d10)[persistent,void]]]",
-                "success": "[[/r 2d10[persistent,void]]]"
+                "criticalSuccess": "@Damage[(2*2d10)[persistent,void]]",
+                "success": "*@Damage[2d10[persistent,void]]"
             },
             "Piercing2d8+8": {
-                "criticalSuccess": "[[/r (2*2d8+16)[persistent,piercing]]]",
-                "success": "[[/r (2d8+8)[persistent,piercing]]]"
+                "criticalSuccess": "@Damage[(2*(2d8+16))[persistent,piercing]]",
```

```suggestion
                "criticalSuccess": "@Damage[(2*2d8+16)[persistent,piercing]]",
```
+16 is already doubled